src/parser/jobs/whm/index.js

```diff
@@ -13,25 +11,14 @@ This analyser aims to identify some of the low-hanging fruit that could be used
 Note that since healing is a very fight-dependent process, it might be that some suggestions do not apply to you, but they may they help to inform you about things you have missed.
 
 If you would like to learn more about WHM, check the guides over at [The Balance](https://thebalanceffxiv.com/), and have a chat in the #whm_questions channel.
+
+Currently this module can track DoT uptime, note when your healing lily gauge is full, detect clipping, detect interrupted casts, and report about missed oGCD casts.
```

Comment:
Clipping is a term that gets overloaded in the community (for overweaving / clipping GCD as well as for re-applying a DOT too soon) -- is there a more clear phrasing we can use here?

Reply:
Not sure, to be honest. I let the WHM mentor drive the wording here.

Reply:
Perhaps detect incorrect weaving?